mono/mini/mini-llvm.c

```diff
@@ -8118,8 +8168,11 @@ emit_method_inner (EmitContext *ctx)
 		}
 	}
 
-	if (!cfg->llvm_only && cfg->compile_aot && mono_threads_are_safepoints_enabled () && requires_safepoint)
```

Comment:
The OP_GC_SAFE_POINT code should be removed.

Reply:
Still used for the LLVM-only mode.

Reply:
Yeah I initially removed it but then Filip reminded me the llvm-only case